tests/app/views/test_services.py

```diff
@@ -78,13 +116,15 @@ def _assert_service_page_url(self):
 
         res = self.client.get('/g-cloud/services/{}'.format(service_id))
         assert_equal(200, res.status_code)
-        assert_true(
-            "<h1>{}</h1>".format(service_title)
-            in res.get_data(as_text=True)
-        )
 
-        self._assert_contact_details(res)
-        self._assert_document_links(res)
+        document = html.fromstring(res.get_data(as_text=True))
+
+        page_title = document.xpath('//div[@id="wrapper"]//h1/text()')[0]
+        assert_equal("{}".format(service_title), page_title)
```

Comment:
Why not just `assert_equal(service_title, page_title)`?

That passes fine with Python 2 - does Python 3 break it or something?